sda/cmd/api/swagger_v1.yml

```diff
@@ -300,6 +300,12 @@ paths:
           schema:
             type: string
           required: true
+        - in: query
+          name: path_prefix
```

Comment:
Should it only be the prefix or can be a substring at any position? I don't think there's big difference in the database query when using `LIKE`.

If we stick to the `prefix`, then the userID (still exist in the filepath) should not be included.

Reply:
That will depend on how you implement the query.
`submission_file_path LIKE '%PREFIX%'` or `submission_file_path LIKE 'PREFIX%'`
The first variant allows you to select *any* substring while that latter is strict for the first part of the path.

Reply:
Sure, you can implement like `submission_file_path LIKE '%PREFIX%'` and in this case `PREFIX` works as a substring. However, if one just read the API specification in Swagger, the name `path_prefix` naturally indicate matching the beginning of the file path, which is misleading. I would suggest change 
`path_prefix` to a less misleading name, e.g. `path_substring` or `path_part`.